src/ol_infrastructure/applications/mitx/__main__.py

```diff
@@ -0,0 +1,45 @@
+from pulumi import StackReference, export
+from pulumi_aws import route53
+
+from ol_infrastructure.components.aws.s3_cloudfront_site import (
+    CloudfrontPriceClass,
+    S3ServerlessSite,
+    S3ServerlessSiteConfig,
+)
+from ol_infrastructure.lib.ol_types import Apps, AWSBase
+from ol_infrastructure.lib.pulumi_helper import parse_stack
+
+fifteen_minutes = 60 * 15
+dns_stack = StackReference("infrastructure.aws.dns")
+stack_info = parse_stack()
+mitx_environment = f"mitx-{stack_info.env_suffix}"
+aws_config = AWSBase(
+    tags={
+        "OU": "mitx",
+        "Environment": mitx_environment,
+        "Application": Apps.mitx_edx.value,
+    }
+)
+
+mitx_zone_id = dns_stack.get_output("mitx_zone_id")
+mfe_app_learning_config = S3ServerlessSiteConfig(
```

Comment:
I think we are going to have to parameterize this from the stack settings to provide for our different "purposes", since I don't think that we can share the compiled MFEs between draft and live for example.

Reply:
Do we even need MFE's to be running in draft?

Reply:
I'd defer to @pdpinch for that one, but we'll want to at least be able to support next vs current as we make that transition (maybe?).

Reply:
Ideally, I'd like to keep things as simple as possible at least until we can get this to work first and then figure out how to handle all the special cases we have, but yeah I'll defer to the two of you

Reply:
Let's just get it running as is and figure it out, like you said.